Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -871,7 +895,99 @@ private void addExtensionTypeSettings(ArrayList<SettingsItem> sl, int wiimoteNum
   {
     switch (extentionType)
     {
-      case 1: // Nunchuk
+      case SettingsActivityPresenter.WIIMOTE_EXT_NONE:
+        final int wiimoteNumberAdjusted;
+
+        if (mSettings.isGameSpecific())
+        {
+          wiimoteNumberAdjusted = wiimoteNumber - 4;
+          mSettings.loadWiimoteProfile(mGameID, wiimoteNumberAdjusted);
+        }
+        else
+        {
+          wiimoteNumberAdjusted = wiimoteNumber - 3;
+        }
+
+        AbstractBooleanSetting wiimoteOrientation = new AbstractBooleanSetting()
+        {
+          Boolean valueSelected;
+
+          @Override
+          public boolean getBoolean(Settings settings)
+          {
+            if (valueSelected != null)
+            {
+              return valueSelected;
+            }
+            else
+            {
+              return getWiimoteSection(wiimoteNumberAdjusted, true)
+                      .getBoolean(SettingsFile.KEY_WIIMOTE_ORIENTATION, false);
+            }
+          }
+
+          @Override
+          public void setBoolean(Settings settings, boolean newValue)
+          {
+            // Set this setting now since multiple settings that change controllers may be modified.
+            SharedPreferences preferences = PreferenceManager
+                    .getDefaultSharedPreferences(mView.getActivity().getApplicationContext());
+            final SharedPreferences.Editor editor = preferences.edit();
+
+            editor.putInt("wiiController", newValue ? InputOverlay.OVERLAY_WIIMOTE_HORIZONTAL :
+                    InputOverlay.OVERLAY_WIIMOTE_VERTICAL);
+            editor.commit();
+
+            getWiimoteSection(wiimoteNumberAdjusted, false).setBoolean(settings.isGameSpecific() ?
+                    SettingsFile.KEY_WIIMOTE_ORIENTATION + wiimoteNumberAdjusted :
+                    SettingsFile.KEY_WIIMOTE_ORIENTATION, newValue);
+
+            valueSelected = newValue;
+            mView.onSettingChanged();
+          }
+
+          @Override
+          public boolean isOverridden(Settings settings)
+          {
+            if (valueSelected != null)
+            {
+              return settings.isGameSpecific() && valueSelected != settings
+                      .getSection(Settings.FILE_WIIMOTE,
+                              Settings.SECTION_WIIMOTE + wiimoteNumberAdjusted)
+                      .getBoolean(SettingsFile.KEY_WIIMOTE_ORIENTATION, false);
```

Comment:
This should use `exists` instead of `getBoolean`, for consistency with how other settings work. Reference: https://github.com/dolphin-emu/dolphin/blob/a7b9e6857b6c4e172a4ec333a6c83d624092cdf3/Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/AbstractLegacySetting.java#L19

Reply:
The problem with that is the custom controller profile has an orientation setting when created, unlike other custom game settings in the GameSettings folder.

Reply:
But that means that the setting actually is overridden, no? So it should show up as overridden.

Reply:
I think I see what was causing me concern. "Extension" and "Orientation" are both Wii profile custom game settings created along with the Wii Profile. This means "Orientation" will always be considered overridden with your suggested change. "Extension" is not considered overridden unless the setting is modified (like the other settings). These settings shouldn't be behaving differently since they come from the same section of the same file. I think I should keep the behavior I introduced, as the `.exists` check seems to assume the setting doesn't exist by default, which isn't true for profile settings.